### PR TITLE
Add December blog post

### PR DESCRIPTION
**Pull request by saragon02**
:heart: Thank you for your contribution!

### Description

> Please describe briefly your pull request.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/howtos/i18n/).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).
    * If you have added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Frontend**

- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines.
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines.
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines.

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Review by mfenner — APPROVED** [review]
The text looks good to me. Minor suggestion: put in the date when the FreiData instance launched or was officially made available.

**Comment by JanLeendertse** [comment]
> The text looks good to me. Minor suggestion: put in the date when the FreiData instance launched or was officially made available.

The release date was 27th of October, 2023.

We expected a slow progress of contributions for the first month. Currently I have several interested people who want to use the APIs.

I am looking into the source text, where to add that detail.

**Comment by JanLeendertse** [comment]
Hi Sara, hi Martin,

I do not have the access rights, and a change of the text requires a fork, which is a bit cumbersome for the slight change. So my suggestion to Martins request:

> When InvenioRDM launched on 27th of October at the University of Freiburg under the label FreiData,

What do you think?

**Comment by mfenner** [comment]
Jan, I like the suggested Text and can put it in the document for you.

Best,

Martin
Am 6. Dez. 2024, 09:52 +0100 schrieb Jan Leendertse ***@***.***>:

**Comment by JanLeendertse** [comment]
> Jan, I like the suggested Text and can put it in the document for you. 
I see that I forgot the year. It should be sorry:

> When InvenioRDM launched on 27th of October 2023 at the University of Freiburg under the label FreiData,

Sorry for the extra loop.

File: content/blog/2024-12-09-december-FreiData/contents.lr [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+team: InvenioRDM
+---
+body:
+
+![](ComputerCenter_UniversityFreiburg.jpg)
+
+Image by Sandra Meyndt/University of Freiburg
+
+
+When InvenioRDM launched at the University of Freiburg under the label [FreiData](https://freidata.uni-freiburg.de/), campus storage options which minted DOIs for publications already existed. For over a decade, Freiburg has hosted an institutional repository called "FreiDok“, which is also used as a documentation system for the university's publication activity. In its policies for handling research data and for Open Science, the university recommends that researchers prioritize visibility to their key target audience as the most important criterion when publishing research data. This often leads researchers to choose externally operated repositories.
```
Comment (saragon02) [inline content/blog/2024-12-09-december-FreiData/contents.lr:16]:
```suggestion
When InvenioRDM launched on 27th of October 2023 at the University of Freiburg under the label [FreiData](https://freidata.uni-freiburg.de/), campus storage options which minted DOIs for publications already existed. For over a decade, Freiburg has hosted an institutional repository called "FreiDok“, which is also used as a documentation system for the university's publication activity. In its policies for handling research data and for Open Science, the university recommends that researchers prioritize visibility to their key target audience as the most important criterion when publishing research data. This often leads researchers to choose externally operated repositories.
```

**Review by mfenner — APPROVED** [review]
LGTM

**Review by carsonicator — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]
LGTM! (the website https://freidata.uni-freiburg.de/ is down at the time of writing this)

**Comment by JanLeendertse** [comment]
> LGTM! (the website https://freidata.uni-freiburg.de/ is down at the time of writing this)

OMG. I will let check this. Thanks for hinting.

**Comment by JanLeendertse** [comment]
Is fixed.

**Comment by saragon02** [comment]
Thanks Jan! The blog is live.

**Comment by JanLeendertse** [comment]
> Thanks Jan! The blog is live.

Thanks for releasing, and for the opportunity to contribute. And for your patience.

**Comment by mfenner** [comment]
Thanks Jan. One minor issue: the cropping of the feature image leads to a blue image. Is that ok, or should I crop the image differently (removing some of the sky on top)?

Bildschirmfoto 2024-12-09 um 21 23 30

**Comment by saragon02** [comment]
Good point, Martin. The preview or thumbnail of the blog's first image is always a little funnily cropped like this. Perhaps there is a larger solution that could be applied to the website?

**Comment by mfenner** [comment]
In my experience (in other blog platforms) the main issue with cropping is different proportions of images. Another issue is that the image is too big (5400 x 3600 px) making loading slow. I also see that several of the latest posts have same issue (lots of blue). Happy to tinker with the image but hope that @JanLeendertse doesn't mind.

**Comment by ntarocco** [comment]
Adding this to the `style.css`, it is a bit better (not perfect):

```css
.blog-list-item-content img {
  width: 100%;
  height: 100%;
  object-fit: cover;
  object-position: center center;
}
```

Screenshot 2024-12-09 at 21 49 37

**Comment by mfenner** [comment]
This looks much better. I also cropped the image to remove some of the blue sky at the top and use a more reasonable resolution (1200 x 620 px). Now trying to bring all this together.

ComputerCenter_UniversityFreiburg

**Pull request merged**